src/EnergyPlus/EvaporativeCoolers.cc

```diff
@@ -2801,7 +2801,7 @@ namespace EvaporativeCoolers {
         auto &EvapCond(state.dataEvapCoolers->EvapCond);
 
         EvapCoolIndex = int(Par(1));
-        DryOrWetOperatingMode = OperatingMode(Par(2));
+        DryOrWetOperatingMode = OperatingMode(int(Par(2)));
```

Comment:
This is fine...for now.  We need a targeted overhaul of the root-finding methods in EnergyPlus at some point.  The ways we packaged up a payload of data was fine in Fortran, but there are much better ways in C++.  In any case, I didn't actually realize we could even compile in VS2017 anymore, so if this is all it takes to get it compiling again, then so be it.  Thanks for the fix.